# Committing increasingly large numbers of files
# 
# reformat_partition: /dev/murphytest03-vg/test
# vcs:                prototype
# vcs_version:        prototype: git version c9baf3a Debug: Fix bad read
# each_file_size:     0x400 bytes (1.0KiB)
# data_gen:           random
# 
# date:          2017-03-25T13:16:36.637180
# commandline:   ./increasing_number_of_files.py prototype 0 8 --mag-step=4 --data-gen=random --tmp=/test --reformat-partition /dev/murphytest03-vg/test
# testversion:   31479dc Dirs: Use fewer dirs in many files test
# hostname:      murphytest03
# platform:      Linux-3.16.0-4-amd64-x86_64-with-debian-8.6
# memtotal:      8064260 kB
# memfree:       925744 kB
# 
# pythonversion:
#     2.7.9 (default, Jun 29 2016, 13:08:31) 
#     [GCC 4.9.2]
# 
# cpuinfo:
#     processor	: 0
#     model name	: Intel(R) Core(TM)2 Duo CPU     E8500  @ 3.16GHz
#     cpu MHz		: 3166.000
#     cache size	: 6144 KB
#     
#     processor	: 1
#     model name	: Intel(R) Core(TM)2 Duo CPU     E8500  @ 3.16GHz
#     cpu MHz		: 3166.000
#     cache size	: 6144 KB
# 
# fsinfo:
#     Filesystem                         Size  Used Avail Use% Mounted on
#     /dev/mapper/murphytest03--vg-test  197G   51G  147G  26% /test
# 
# diskinfo:
#     sda	vendor: ATA, model: ST3250318AS	scheduler: noop deadline [cfq]
# 
# Command results:
# ok:      Command completed successfully
# no_exec: Command was never executed
# failed:  Command failed
# 
# Verification results:
# ver_err:  Could not verify due to error during verification
# no_ver:   Verification was never performed
# bad:      Verification discovered an error
# verified: Verified OK
# assumed:  Assumed ok because dependent commands successful
# 
mag     filecount    totalbytes   cre_time    c1_time       c1_size   c1_cmd    c1_ver   c1_repo     c1_user     c1_nice   c1_system     c1_idle   c1_iowait  stat1_time  stat1_cmd  stat2_time  stat2_cmd    c2_time       c2_size   c2_cmd    c2_ver   c2_repo     c2_user     c2_nice   c2_system     c2_idle   c2_iowait  cleanup_time
# CallFailedError(cmd='sudo umount /dev/murphytest03-vg/test', exitcode='32')
  6       7500000  0x01c9c38000    967.892  20210.608        (None)   failed       bad  verified       53503           0       74772     1379313     2515573      (None)    no_exec      (None)    no_exec     (None)        (None)  no_exec    no_ver    no_ver      (None)      (None)      (None)      (None)      (None)         0.500
# CallFailedError(cmd='sudo umount /dev/murphytest03-vg/test', exitcode='32')
  7      10000000  0x02625a0000     (None)     (None)        (None)  no_exec    no_ver    no_ver      (None)      (None)      (None)      (None)      (None)      (None)    no_exec      (None)    no_exec     (None)        (None)  no_exec    no_ver    no_ver      (None)      (None)      (None)      (None)      (None)         0.021
# CallFailedError(cmd='sudo umount /dev/murphytest03-vg/test', exitcode='32')
  7      25000000  0x05f5e10000     (None)     (None)        (None)  no_exec    no_ver    no_ver      (None)      (None)      (None)      (None)      (None)      (None)    no_exec      (None)    no_exec     (None)        (None)  no_exec    no_ver    no_ver      (None)      (None)      (None)      (None)      (None)         0.015
# CallFailedError(cmd='sudo umount /dev/murphytest03-vg/test', exitcode='32')
  7      50000000  0x0bebc20000     (None)     (None)        (None)  no_exec    no_ver    no_ver      (None)      (None)      (None)      (None)      (None)      (None)    no_exec      (None)    no_exec     (None)        (None)  no_exec    no_ver    no_ver      (None)      (None)      (None)      (None)      (None)         0.014
# CallFailedError(cmd='sudo umount /dev/murphytest03-vg/test', exitcode='32')
  7      75000000  0x11e1a30000     (None)     (None)        (None)  no_exec    no_ver    no_ver      (None)      (None)      (None)      (None)      (None)      (None)    no_exec      (None)    no_exec     (None)        (None)  no_exec    no_ver    no_ver      (None)      (None)      (None)      (None)      (None)         0.014
